docker/build.sh

```diff
@@ -36,12 +38,25 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 
 cd ${SCRIPT_DIR}/..
 
-docker build -t ${GHCR}:base-${TARGET} \
+docker build -t ${GHCR}-common:base-${TARGET} \
```

Comment:
@Binyang2014 Why is this needed?

Reply:
Due to need to install rccl, will check AMD guys for image supported.